app/server/index.ts

```diff
@@ -46,6 +47,8 @@ app.prepare().then(async () => {
 
   server.use(await ssoMiddleware());
 
+  server.get("/auth-callback", createUserMiddleware("localhost", port));
```

Comment:
Refactoring idea: sso-express would export its configuration or default configuration, so we don't have to use a magic string here